CHANGELOG.md

```diff
@@ -12,11 +12,35 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Build-time environment variable `CONCORDIUM_WALLET_CRYPTO_FRAMEWORK_PATH`
   for resolving the framework to a path instead of using the released one.
 
+- Functions and associated types for creating identity issuance and recovery requests as well as account credential (deployment).
+
+  UDL signatures:
+  
+  - `string identity_issuance_request_json(IdentityIssuanceRequestParameters params)`
+  - `string identity_recovery_request_json(IdentityRecoveryRequestParameters params)`
+  - `AccountCredentialResult account_credential(AccountCredentialParameters params)`
+  - `string account_credential_deployment_hash_hex(AccountCredential credential, u64 expiry_unix)`
+  - `string account_credential_deployment_signed_payload_hex(SignedAccountCredential credential)`
+  
+  The parameter types mirror the corresponding "input" types from `wallet_library` but use only types supported by UniFFI.
+  The values are translated into these library types via JSON encoding/decoding.
+  This is in contrast to the Java SDK where the value is passed as a JSON encoded string
+  which is then decoded directly into the library input type.
+  Doing it this way ensures that type unsafe conversions happen internally in this library where it's easily tested
+  rather than at the FFI boundary.
```

Comment:
I don't understand what difference this is pointing out since the FFI boundary is hidden in the Java SDK as well, in the internals of the library.